library/src/main/assets/google_map.html

```diff
@@ -268,6 +268,23 @@
   return '#' + ('000000' + (0xFFFFFF & color).toString(16)).slice(-6);
 }
 
+function addGeoJsonLayer(jsonString, strokeWidth, strokeColor, fillColor) {
```

Comment:
JavaScript is not a statically typed language so it wont let us specify types for args. I'll add some comments though